metron-platform/metron-test-utilities/src/main/java/org/apache/metron/test/mock/MockHTable.java

```diff
@@ -437,15 +463,14 @@ public ResultScanner getScanner(byte[] family, byte[] qualifier)
     return getScanner(scan);
   }
 
-  List<Put> putLog = new ArrayList<>();
```

Comment:
Oh, sorry guys.  Ignore me.  You're talking Collections.synchronizedList, not Guava's synchronized supplier.  Yeah, I think you're right.